### ISSUE TITLE
Feature Request: only check specific rules

### ISSUE THREAD

**Issue by 304NotModified** [hadolint/hadolint#1156]
Right now, hadolint checks all rules by default. If you want to use only specific rules, you have to exclude everything else:

```yaml
ignored:
  - DL3000
  - DL3001
  - DL3002
  - DL3003
  - DL3004
  - DL3005
  # TODO ... all rules
```

**Why this is problematic:**
- Every hadolint update adds new rules that can suddenly break CI/CD
- Maintaining a massive exclusion list becomes tedious
- The config doesn't show which rules I actually care about - only what I don't want

## What I'd Like Instead

Just let me specify which rules I **want** to check:

```yaml
only:
  - DL3001
  - DL3007
  - DL3008
```

**Or as alternative: presets**

```yaml
# Start with a sensible baseline
default: recommended  # e.g. all, none, security-focused etc

# Then tweak it
include:
  - DL3025
exclude:
  - DL3007  # We're not ready for this one yet
```

## Why This Would Be Better

- **No more surprise failures**: New rules don't randomly break builds
- **Actually readable config**: 5 lines instead of 50+
- **Clear intent**: Everyone sees what we care about
- **Easy onboarding**: Start with a few rules, add more as we go

**Comment by 304NotModified** [comment]
FYI in this validator (https://github.com/304NotModified/SLNX-validator) I had the same issue and I fixed it by allowing *, e.g.

```bash
# Set everything to INFO, but keep SLNX011 as MAJOR
slnx-validator MySolution.slnx --info * --major SLNX011

# Ignore everything except SLNX013
slnx-validator MySolution.slnx --ignore * --major SLNX013
```
